crates/dbsp/src/operator/dynamic/controlled_filter.rs

```diff
@@ -271,7 +271,7 @@ where
             cursor.step_key();
         }
 
-        unsafe { self.error_stream_val.put(errors.done()) };
+        self.error_stream_val.put(errors.done());
```

Comment:
this change is surprising

Reply:
it used to be unsafe, but now is safe :)